source/documentation/tools/index.md

```diff
@@ -0,0 +1,34 @@
+# Tools
+
+## RStudio and JupyterLab
+
+Analytical Platform comes with tools including:
+
+* RStudio - a development environment for writing R code and R Shiny apps
+* JupyterLab - see [JupyterLab section](../jupyterlab.html)
+* Airflow - see [Airflow section](../airflow.html)
+
+## Managing your analytical tools
+
+Tools on Analytical Platform include RStudio, JupyterLab and Airflow sandbox. To use these, each user must 'start' their own copy of the software. This gives them the benefit of reserved memory space (compared to the more common shared R Studio Server), and some control over the version of R that is running.
+
+To manage your tools:
+
+1. Go the Analytical Platform [control panel](https://controlpanel.services.alpha.mojanalytics.xyz).
+2. Select the __Analytical tools__ tab.
+
+Use the buttons shown against each tool to manage your copy of the tool:
+
+* "Deploy" - The tool is not yet deployed - this is the initial state. You need to "Deploy" to be able to use the tool for the first time. It sets you up with the latest version and starts it. This may few minutes.
+* "Open" - The tool is either "Idled" (configured but not running) or "Ready" (running). If your RStudio, JupyterLab or Airflow instance is inactive on a Tuesday evening it will be idled. Press "Open" to navigate to the tool in your browser, and if it is not running it will start it (run or "unidle" it). Starting a tool usually takes about 30 seconds, but occasionally will take a few minutes.
+* "Restart" - Often problems with the tool can be solved by restarting the software on the server.
+* "Upgrade" - Another release of the tool is available. Occasionally new versions of R Studio are made available on the Analytical Platform. In this case all users will be given the opportunity to upgrade on the control panel. New versions provide new features and bugfixes to the tool. In addition, there some releases come with improvements to the way RStudio is containerized and integrated with Analytical Platform. You should aim to upgrade when it is offered, although just in case it causes minor incompatibilities with your R code, you should not do it in the days just before you have a critical release of your work. When pressed, the status will change to 'Deploying' and then 'Upgraded'. The __Upgrade__ button will no longer be visible (until another version becomes available).
+
+### RShiny
+
+To use conda in RShiny applications, you will need to use a different `Dockerfile` to deploy the app. The [conda branch of the rshiny-template](https://github.com/moj-analytical-services/rshiny-template/tree/conda) has an appropriate `Dockerfile` for this purpose. This is also necessary if you wish to use Python in your Shiny application, including using `dbtools` for accessing Amazon Athena databases.
+
+### Analytical Platform limitations
+
+There are a number of limitations and pitfalls to conda management to be aware of.
```

Comment:
This probably either needs a fully description of what these limitations and pitfalls are or a link to a section on it.

Reply:
Whilst I agree whole heartily with this, this was a copy pasta fro the old layout, so I think we should address this in another PR.

Reply:
I think previously it had a section below detailing one of the limitations.

https://user-guidance.services.alpha.mojanalytics.xyz/tools.html#analytical-platform-limitations

i.e. this section is the limitation
https://user-guidance.services.alpha.mojanalytics.xyz/tools.html#r-package-versions-on-conda

So perhaps you could move this back to be below this text?

Reply:
Ah I know what happened here. I moved it given there wasn't a whole lot of context of what it was talking about. I think this really clearly demonstrates how organic this documentation is.

Reply:
I've moved the one section back but I really think it needs a proper going over in the next few weeks.